build.sbt

```diff
@@ -213,6 +213,31 @@ lazy val contribs = (project in file("contribs"))
     Compile / compile := ((Compile / compile) dependsOn createTargetClassesDir).value
   ).configureUnidoc()
 
+lazy val deltaSharing = (project in file("deltasharing"))
+  .dependsOn(spark % "compile->compile;test->test;provided->provided")
+  .settings(
+    name := "delta-sharing-spark",
+    commonSettings,
+    scalaStyleSettings,
+    javaOnlyReleaseSettings,
```

Comment:
@vkorukanti @linzhou-db can you validate that the maven artifact name is as expected? no scala version in it?

Reply:
no it's not intended, removed `javaOnlyReleaseSettings`.